doc/usermanagment/user-managment.md

```diff
@@ -94,3 +94,62 @@ To delete user from the cluster, remove userRef from "spec", then use following
 ```console
 kubectl apply -f redis.yaml
 ```
+
+## Kafka user certificates
+
+### Creation flow
+If you want to use MTLS for a Kafka user you should create `UserCertificate` resource.
```

Comment:
If you want to use MTLS for a Kafka user you should create a `UserCertificate` resource.

Reply:
Fixed